.gitlab/e2e/e2e.yml

```
@@ -294,6 +294,7 @@ new-e2e-aml:
     - deploy_windows_testing-a7
     - qa_agent
     - qa_agent_jmx
+    - qa_agent_fips_jmx
```

I think it should be this option:
```suggestion
    - qa_agent_fips_jmx
```
https://github.com/DataDog/datadog-agent/blob/dbadf918107b26e1733db87a51801f0f79893ff8/.gitlab/dev_container_deploy/e2e.yml#L37C1-L37C18